Dockerfile

```diff
@@ -73,6 +73,13 @@ WORKDIR /tmp/nemo
 COPY requirements .
 RUN for f in $(ls requirements/*.txt); do pip install --disable-pip-version-check --no-cache-dir -r $f; done
 
+# install quantization support
+RUN git clone https://github.com/NVIDIA/TensorRT.git && \
```

Comment:
@skyw do you want to take a specific commit here. Or you are confident that TOT will be fine?

Reply:
TOT should be fine. It is only updated once a month, together with other DLFW update.